corehq/apps/cloudcare/templates/cloudcare/partials/users/user_row.html

```diff
@@ -4,7 +4,7 @@
       <i class="fa fa-user module-icon" aria-hidden="true"></i>
     </div>
   </td>
-  <td class="module-column">
+  <td class="module-column module-column-name">
```

Comment:
The `module-column-name` class gets added here but it looks like the css class was removed, is that right? Or does it serve some other purpose as well?

Reply:
I had removed it to get rid of the relative positioning that caused issues. With this commit I am adding it back in because it missing caused some other issue in app preview. Instead I removed the property from the css class and added it to the direct parent of the icon tags that needed the relative position.